app/src/main/java/com/flexcode/devspace/core/activities/MainActivity.kt

```diff
@@ -40,18 +42,16 @@ class MainActivity : AppCompatActivity() {
 
         binding.bottomNavigationView.apply { setupWithNavController(navController) }
 
-
-        //hide bottom nav view
-        navController.addOnDestinationChangedListener { _, destination, _ ->
-            when (destination.id) {
-                R.id.settingsFragment -> binding.bottomNavigationView.isGone = true
-                R.id.followersFragment -> binding.bottomNavigationView.isGone = true
-                R.id.followingFragment -> binding.bottomNavigationView.isGone = true
-                R.id.welcomeFragment -> binding.bottomNavigationView.isGone = true
-                else -> binding.bottomNavigationView.isVisible = true
+        // Hides the bottom navigation view
+        lifecycleScope.launchWhenResumed {
```

Comment:
thanks a lot for your contributions I always find them so insightful and also help me learn and improve myself as a developer.